### ISSUE TITLE
Starting object store with directory /dev/shm and huge page support disabled

### ISSUE THREAD

**Issue by invinciberry** [blakeblackshear/frigate#140]
When the docker starts I get
`/arrow/cpp/src/plasma/store.cc:1226: Allowing the Plasma store to use up to 0.4GB of memory.
/arrow/cpp/src/plasma/store.cc:1253: Starting object store with directory /dev/shm and huge page support disabled`
I am not sure if I am doing the shm_size setting correctly, I have it installed on Unraid with Extra Parameters "--shm-size=1024m".

**Comment by blakeblackshear** [comment]
Those are normal messages when it is working.

**Comment by invinciberry** [comment]
Sure. Also, I noticed when I set take_frame to 1 for a 30fps camera. The debug preview will have a much higher delay compare with take_frame=2. I am not sure where the delay is created since I am using a coral which in theory can process 100+ frames/s.

**Comment by blakeblackshear** [comment]
What hardware are you running on? What inference speeds are you seeing on the /debug/stats endpoint?

**Comment by invinciberry** [comment]
I am running it on an unraid box with ryzen 3600x. My inference speed is about 24.

**Comment by blakeblackshear** [comment]
With inference speeds of 24ms you will max out at about 40fps (1000ms/24). Your USB3 bus speeds are probably reduced by a virtualization layer somewhere and you are not sending data to the coral as fast as it can receive it. An ideal setup should show inference speeds of about 10ms.

**Comment by simonbirt** [comment]
One reason for a lower inference speed _could_ be that you have the Coral plugged into a USB 2.0 port instead of a USB 3 one. My motherboard has both. You should be able to tell from the output of `lsusb`. When I moved my Coral from the 'wrong' port to the 'right' one, my inference speed went from approx 25 to approx 8.

**Comment by invinciberry** [comment]
Any ideas on how I can improve that? I am running coral on docker instead of a Linux VM. And I think I used a 10Gbs USB 3.0 port but I will double-check later tonight. Here is the output of lsusb.
`Bus 006 Device 002: ID 0781:5581 SanDisk Corp. Ultra
Bus 006 Device 001: ID 1d6b:0003 Linux Foundation 3.0 root hub
Bus 005 Device 001: ID 1d6b:0002 Linux Foundation 2.0 root hub
Bus 004 Device 001: ID 1d6b:0003 Linux Foundation 3.0 root hub
Bus 003 Device 002: ID 0b05:18f3 ASUSTek Computer, Inc. AURA LED Controller
Bus 003 Device 001: ID 1d6b:0002 Linux Foundation 2.0 root hub
Bus 002 Device 001: ID 1d6b:0003 Linux Foundation 3.0 root hub
Bus 001 Device 003: ID 18d1:9302 Google Inc. 
Bus 001 Device 001: ID 1d6b:0002 Linux Foundation 2.0 root hub`

**Comment by simonbirt** [comment]
`Bus 001 Device 003: ID 18d1:9302 Google Inc. 
Bus 001 Device 001: ID 1d6b:0002 Linux Foundation 2.0 root hub`

The above seems to indicate that the Coral is connected to Bus 1 which is USB 2.0.  Your USB 3 Bus IDs are 2,4 and 6. The only thing currently connected to any of the USB3 hubs is a Sandisk device.

**Comment by invinciberry** [comment]
Interesting, I will change the port later tonight. Can it be a cable problem? I am assuming the cable that came with coral is USB 3.0.

**Comment by simonbirt** [comment]
I don't think it's a cable issue. I too am using the cable that came with the Coral.

**Comment by invinciberry** [comment]
So I was able to solve the issue by switching to another usb port.

I found out when coral is plugged into a teal blue USB 3.0 10Gbps (USB 3.2 Gen2) port it will only run at USB 2.0 speed (~24ms latency), and when plugged into a normal dark blue USB 3.0 (USB 3.2 Gen1) port it will run at normal 3.0 speed (~8ms latency). I suspect it's a Linux driver issue, but it doesn't really matter anyway since I have plenty normal dark blue USB 3.0 (USB 3.2 Gen 1) ports.

Man I hate those new USB naming scheme.
